### PR TITLE
Added automatic each serializer

### PR DESCRIPTION
**Pull request by hdf1986**
# Summary
- Added conditional to detect if activemodelseriazers is present and use the serializers defined for the models

# Issue
Solves #47

### CODE REVIEW

**Comment by hdf1986** [comment]
@alanhala It's already there, in the first example we don't use each_serializer, and that wasn't possible before this pr, so we are fixing to make it behave like the current readme :P

File: lib/wor/paginate/formatter.rb [hunk 1]
```diff
@@ -32,8 +32,13 @@ def paginated_content
       end
 
       def serialized_content
-        return paginated_content.as_json if serializer.blank?
-        paginated_content.map { |item| serializer.new(item) }
+        return paginated_content.map { |item| serializer.new(item) } if serializer.present?
```
Comment (alanhala) [inline lib/wor/paginate/formatter.rb]: I think this is not responsibility of the formatter. The formatter should only build the json response. What if we create our serializer that checks this condition and the formatter only talks with its interface without knowing that it has to check if the user has sent a serializer or not?

File: lib/wor/paginate/formatter.rb [hunk 2]
```diff
@@ -32,8 +32,13 @@ def paginated_content
...
+
+        if defined? ActiveModelSerializers::SerializableResource
+          ActiveModelSerializers::SerializableResource.new(paginated_content).as_json
```
Comment (alanhala) [inline lib/wor/paginate/formatter.rb]: Have you tested what happens if the gem can't find a serializer for the paginated_content? does it raise an exception?

**Comment by alanhala** [comment]
Add some specs for this new feature please

**Pull request closed without merging**